CHANGELOG.md

```diff
@@ -5,7 +5,11 @@
 
 ### Fixed
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
+<<<<<<< HEAD
```

Comment:
@jedelbo You forgot to remove these lines.

Reply:
Damn, I forgot to press "save" en the editor. Will be fixed in the release PR.